tests/test_cli.py

```diff
@@ -72,12 +75,12 @@
     "[2020-08-29] AAF035 (3).jpg",
     "[2020-08-29] AAF035.jpg",
     "DSC03584.dng",
-    "Frítest (1).jpg",
-    "Frítest (2).jpg",
-    "Frítest (3).jpg",
-    "Frítest_edited (1).jpeg",
-    "Frítest_edited.jpeg",
-    "Frítest.jpg",
+    "Frítest (1).jpg",
+    "Frítest (2).jpg",
+    "Frítest (3).jpg",
+    "Frítest_edited (1).jpeg",
+    "Frítest_edited.jpeg",
+    "Frítest.jpg",
```

Comment:
This causes most export tests to fail on macOS.  The unicode for macOS must be NFD or anyone who's previously exported unicode filenames with osxphotos will have their exports corrupted (duplicate exports, invalid updates, etc).  These file names are chosen because they came from users who had issues with exporting in unicode. For tests to pass on both macOS and linux you'll need separate versions of the test filenames. Can linux worth with NFD filenames? If so, perhaps it would be better to leave everything as NFD on export.

Reply:
Hi @dvdkon I would love to get this merged but need to get the unicode names in tests fixed. If you're able to add logic to the tests to use the right unicode names depending on platform I can get this merged. Perhaps it might be possible to transform the names from NFC/NFD when importing the test module so you don't need two separate sets of names.

Reply:
Hi, sorry for not getting to this sooner. It would be best to leave Unicode issues to the user and continue as on macOS, but when running on Linux without the changes I've made, many tests fail, since somehow `osxphotos` is working with both NFC and NFD versions of the same strings. My guess is the Photos DB is not properly normalised, since it doesn't matter on HFS+.

I think your idea of re-normalising the filenames on import based on the platform is the best approach forward. It's going to be strict in requiring that it uses NFD on macOS and NFC elsewhere, but that might be for the better. I'll rebase on master and make the changes soon.